lib/features/login/data/datasource_impl/hive_account_datasource_impl.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:get/get.dart';
+import 'package:model/account/account.dart';
+import 'package:tmail_ui_user/features/login/data/datasource/account_datasource.dart';
+import 'package:tmail_ui_user/features/login/data/local/account_cache_manager.dart';
+
+class HiveAccountDatasourceImpl extends AccountDatasource {
+  final AccountCacheManager _accountCacheManager = Get.find<AccountCacheManager>();
```

Comment:
I think you shouldn't directly call it like that. You should pass it in `constructor` then call it in `bindings`